buildconfig/class_maker.py

```diff
@@ -18,6 +18,11 @@
 
 VERSION = "1.0"
 
+#======================================================================
+def get_year():
+    """returns the current year"""
+    return datetime.datetime.now().date().year
```

Comment:
`.date()` is unnecessary. `datetime.datetime.now().year` works. This is too minor a change to worry about changing imo, just an fyi